addons/mass_mailing/views/mailing_list_views.xml

```diff
@@ -192,6 +193,86 @@
                                 </div>
                             </div>
                         </div>
+                        <!-- Card when the Kanban view is not grouped -->
+                        <div class="oe_kanban_global_click o_mailing_list_kanban_ungrouped w-100 m-0 d-flex flex-row justify-content-between align-items-center flex-wrap">
+                            <div class="col-lg-3 col-sm-6 col-12 py-0 my-auto">
+                                <div class="d-flex text-large">
+                                    <span class="d-inline-block text-truncate font-weight-normal text-large"
+                                          t-att-title="record.name.value">
+                                        <field name="name"/>
+                                    </span>
+                                </div>
+                            </div>
+                            <div class="o_mailing_list_kanban_counts col-lg-1 col-12 my-auto d-flex">
+                                <div class="d-flex flex-row justify-content-start justify-content-md-end">
+                                    <div class="my-auto p-0">
+                                        <button name="action_view_contacts" type="object"
+                                            class="o_mailing_list_kanban_button o_mailing_list_kanban_big_nb text-primary font-weight-bold">
+                                            <field name="contact_count"/>
+                                        </button>
+                                   </div>
+                                    <div class="my-auto px-0">
+                                        <button name="action_view_contacts" type="object"
+                                            class="o_mailing_list_kanban_button text-large text-left ml-2">
+                                            Total <br/>Contacts
+                                        </button>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="o_mailing_list_kanban_stats col-lg-5 col-sm-12 col-12 py-0 my-3 my-sm-auto d-flex justify-content-sm-between justify-content-start flex-wrap">
+                                <a class="mr-sm-0 mr-3 text-large" tabindex="-1"
```

Comment:
Looked a bit at the `tabindex="-1"`, did not know it.

Do we need this here? It is some kind of good practice?

Reply:
it's needed because a picky po didn't want to be able to select them with tab :)